pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -517,18 +517,31 @@ pub mod pallet {
 					domain_address,
 					message,
 					router_id,
-				} => Self::process_inbound_message(domain_address, message, router_id),
+				} => {
+					let mut counter = 0;
+
+					let res = Self::process_inbound_message(
+						domain_address,
+						message,
+						router_id,
+						&mut counter,
+					);
+
+					let weight = match counter {
+						0 => LP_DEFENSIVE_WEIGHT / 10,
```

Comment:
We want to add just a bit if we fail even before starting to process the first message

Reply:
Should we maybe add a `LP_MIN_WEIGHT` or something?

Reply:
I would prefer hardcoding that as well, yes. An alternative would be a simple `fn defensive_weight(counter)`.

Reply:
I didn't read it, and I had the auto-merge enabled. Tech-debt